PROJECT

```diff
@@ -0,0 +1,13 @@
+domain: nodefeaturediscoveries.nfd.kubernetes.io
+layout: go.kubebuilder.io/v3
+projectName: node-feature-discovery
```

Comment:
does this refer to the operand or the operator?

Reply:
missed it on yesterday pass